### PR TITLE
Handle Mongo API requests in microservice mode

### PR DESCRIPTION
**Pull request by ilia-tsyplenkov**
## Description

Handle Mongo API requests in microservice way (for microservice mode)

## Type of change
Mongo used ExecutorCommands instance directly. Now it uses same instance, but as a part of Executor instance

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: mindsdb/api/mongo/classes/query_sql.py [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+
+    def __init__(self, **kwargs):
+        for arg in kwargs:
+            setattr(self, arg, kwargs[arg])
 
 
 def run_sql_command(mindsdb_env, ast_query):
     sql_session = SessionController()
     sql_session.database = 'mindsdb'
+    sqlserver = SqlServerStub(connection_id=-1)
```
Comment (ea-rus) [inline mindsdb/api/mongo/classes/query_sql.py:20]: sql server in mongo server. not good

File: mindsdb/api/mongo/classes/query_sql.py [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
 
-    command_executor = ExecuteCommands(sql_session, executor=None)
-    ret = command_executor.execute_command(ast_query)
-    if ret.error_code is not None:
-        raise Exception(ret.error_message)
+    executor = Executor(sql_session, sqlserver)
+    executor.query_execute(ast_query.to_string())
```
Comment (ea-rus) [inline mindsdb/api/mongo/classes/query_sql.py:23]: it doesn't keep datatypes, for example datetime becomes string.
need to implement ast serialize/deserialize

Comment (ilia-tsyplenkov) [reply mindsdb/api/mongo/classes/query_sql.py]: Fixed

File: mindsdb/api/mongo/classes/query_sql.py (hunk 1 above)
Comment (ilia-tsyplenkov) [reply mindsdb/api/mongo/classes/query_sql.py:20]: Let me rephrase: sqlserver in Executor is not a good stuff at all, right?
